website/docs/guides/dbt-python-bigframes.md

```diff
@@ -0,0 +1,384 @@
+```
+
+5. **Create a GCS bucket to hold the logs**
+```python
+#Create GCS bucket
+gcloud storage buckets create gs://${GOOGLE_CLOUD_PROJECT}-bucket-logs --location=${REGION}
+#Grant Storage Admin over the bucket to your SA 
+gcloud storage buckets add-iam-policy-binding gs://${GOOGLE_CLOUD_PROJECT}-bucket-logs --member=serviceAccount:dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com --role=roles/storage.admin
+```
+
+## Client Side Configuration 
+[Not required in dbt Cloud]
+
+This is the configuration of the server where the dbt python package will be installed (e.g. local dev environment, GCE, GKE ..)
+
+1. **Install dbt-core and the dbt-bigquery adapter**
+
+NOTE: This assumes bigframes support is already available on the dbt-bigquery package
+
+```python
+pip install dbt-core dbt-bigquery
+```
+2. **Generate a service account JSON key**
+This guide uses the service account JSON Key as auth method, which is the preferred method for production environments. 
+[Alternatives found here.](https://docs.getdbt.com/docs/core/connect-data-platform/bigquery-setup#authentication-methods)
+```python
+#Generate the JSON key
+gcloud iam service-accounts keys create dbt-bigframes-sa-key.json --iam-account=dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com
+```
+
+3. **Start and configure a new dbt project**
+```python
+dbt init
+```
+Enter values on the interactive shell:
+- Enter dbt project name. 
+- Select bigquery as database [1]
+- Select service account as authentication method [2]
+- Enter path of the JSON keyfile
+- Enter GCP project name. 
+- Enter BQ dataset 
+- Leave threads to 1 for development purposes
+- Leave timeout to the default value of 300 
+- Select location e.g. us-central1
+
+4. **Edit dbt profile.yml under $HOME/.dbt
+
+For example (`service account` credential)
+
+```yaml
+my_dbt_project_sa:
+  outputs:
+    dev:
+      dataproc_region: us-central1
+      dataset: jialuo_test_us
+      gcs_bucket: jialuo_bf_dbt_integration_dev1
+      job_execution_timeout_seconds: 300
+      job_retries: 1
+      keyfile: /usr/local/google/home/jialuo/dbt_t2/secrets/sa_key.json
+      location: US
+      method: service-account
+      priority: interactive
+      project: bigframes-dbt-integration-dev
+      threads: 1
+      type: bigquery
+  target: dev
+```
+
+OR ('Oauth' credential)
+
+```yaml
+my_dbt_project:
+  outputs:
+    dev:
+      dataproc_region: us-central1
+      dataset: jialuo_test_us
+      gcs_bucket: jialuo_bf_dbt_dev2
+      job_execution_timeout_seconds: 300
+      job_retries: 1
+      location: US
+      method: oauth
+      priority: interactive
+      project: bigframes-dev-perf
+      threads: 1
+      type: bigquery
+  target: dev
+```
+Please note, the bigframes submission method does **NOT** introduce any new configuration in profile.yml under $HOME/.dbt.
+
+5. Set Bigframes submission method
```

Comment:
I would move this submission piece into the next section so you can explicitly tell them this is how to execute a python model.

All that you need to say is there's two ways to configure this - one the project.yml level or in the model config.